code/parachain/frame/staking-rewards/src/test/mod.rs

```diff
@@ -246,23 +236,17 @@ fn create_staking_reward_pool_should_fail_when_slashed_minimum_amount_is_less_th
 					end_block: 5,
 					reward_configs: default_reward_config(),
 					lock: LockConfig {
-						duration_presets: [
-							(
-								ONE_HOUR,
-								FixedU64::from_rational(101, 100)
-									.try_into_validated()
-									.expect("valid reward multiplier")
-							), // 1%
-							(
-								ONE_MINUTE,
-								FixedU64::from_rational(11, 10)
-									.try_into_validated()
-									.expect("valid reward multiplier")
-							), // 0.1%
-						]
-						.into_iter()
-						.try_collect()
-						.unwrap(),
+						duration_multipliers: bounded_btree_map! {
+							// 1%
+							ONE_HOUR => FixedU64::from_rational(101, 100)
+								.try_into_validated()
+								.expect(">= 1"),
```

Comment:
is the old error message not valid anymore?

Reply:
I read it as "expect <the value> to be >= 1", which is easier to grok than "expect <the value> to be a valid reward multiplier". just a semantic thing, doesn't really mean anything since it's a test anyways